pkg/sdk/databases.go

```diff
@@ -231,11 +258,21 @@ func (v *databases) CreateShared(ctx context.Context, id AccountObjectIdentifier
 
 // CreateSecondaryDatabaseOptions is based on https://docs.snowflake.com/en/sql-reference/sql/create-database.
 type CreateSecondaryDatabaseOptions struct {
-	create                  bool                     `ddl:"static" sql:"CREATE"`
-	database                bool                     `ddl:"static" sql:"DATABASE"`
-	name                    AccountObjectIdentifier  `ddl:"identifier"`
-	primaryDatabase         ExternalObjectIdentifier `ddl:"identifier" sql:"AS REPLICA OF"`
-	DataRetentionTimeInDays *int                     `ddl:"parameter" sql:"DATA_RETENTION_TIME_IN_DAYS"`
+	create                     bool                     `ddl:"static" sql:"CREATE"`
+	OrReplace                  *bool                    `ddl:"keyword" sql:"OR REPLACE"`
+	Transient                  *bool                    `ddl:"keyword" sql:"TRANSIENT"`
+	database                   bool                     `ddl:"static" sql:"DATABASE"`
+	IfNotExists                *bool                    `ddl:"keyword" sql:"IF NOT EXISTS"`
+	name                       AccountObjectIdentifier  `ddl:"identifier"`
+	primaryDatabase            ExternalObjectIdentifier `ddl:"identifier" sql:"AS REPLICA OF"`
+	DataRetentionTimeInDays    *int                     `ddl:"parameter" sql:"DATA_RETENTION_TIME_IN_DAYS"`
+	MaxDataExtensionTimeInDays *int                     `ddl:"parameter" sql:"MAX_DATA_EXTENSION_TIME_IN_DAYS"`
+	ExternalVolume             *AccountObjectIdentifier `ddl:"identifier,equals" sql:"EXTERNAL_VOLUME"`
+	Catalog                    *AccountObjectIdentifier `ddl:"identifier,equals" sql:"CATALOG"`
+	DefaultDDLCollation        *string                  `ddl:"parameter,single_quotes" sql:"DEFAULT_DDL_COLLATION"`
+	LogLevel                   *LogLevel                `ddl:"parameter,single_quotes" sql:"LOG_LEVEL"`
+	TraceLevel                 *TraceLevel              `ddl:"parameter,single_quotes" sql:"TRACE_LEVEL"`
+	Comment                    *string                  `ddl:"parameter,single_quotes" sql:"COMMENT"`
```

Comment:
No Tag?

Reply:
No, because secondary databases are read-only (I think it's somewhere in the usage notes in the create database documentation).